code/modules/research/designs/autolathe_designs.dm

```diff
@@ -617,4 +609,132 @@
+	materials = list(MAT_METAL = 5000)
+	build_path = /obj/item/weapon/tank/internals/oxygen
+	category = list("initial", "Atmos")
+
+/datum/design/yellowtank
+	name = "Yellow oxygen tank"
+	id = "yellowtank"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 5000)
+	build_path = /obj/item/weapon/tank/internals/oxygen/yellow
+	category = list("initial", "Atmos")
+
+/datum/design/redtank
+	name = "ed oxygen tank"
+	id = "redtank"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 5000)
+	build_path = /obj/item/weapon/tank/internals/oxygen/red
+	category = list("initial", "Atmos")
+
+/datum/design/anesthetictank
+	name = "Anesthetic tank"
+	id = "anesthetictank"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 5000)
+	build_path = /obj/item/weapon/tank/internals/anesthetic
+	category = list("initial", "Atmos")
+
+/datum/design/plasmatank
+	name = "Plasma tank"
+	id = "plasmatank"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 5000)
+	build_path = /obj/item/weapon/tank/internals/plasma
+	category = list("initial", "Atmos")
+
+/datum/design/extinguisher
+	name = "Fire extinguisher"
+	id = "extinguisher"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 500)
+	build_path = /obj/item/weapon/extinguisher
+	category = list("initial", "Atmos")
+
+/datum/design/miniextinguisher
+	name = "Pocket fire extinguisher"
+	id = "miniextinguisher"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 300)
+	build_path = /obj/item/weapon/extinguisher/mini
+	category = list("initial", "Atmos")
+
+/datum/design/pipepainter
+	name = "Pipe painter"
+	id = "pipepainter"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 5000, MAT_GLASS = 2000)
+	build_path = /obj/item/device/pipe_painter
+	category = list("initial", "Atmos")
+
+/datum/design/firehelmet
+	name = "Firefighter helmet"
+	id = "firehelmet"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 1000, MAT_GLASS = 200)
+	build_path = /obj/item/clothing/head/hardhat/red
+	category = list("initial", "Atmos")
+
+/datum/design/watertank
+	name = "Backpack firefighter tank"
+	id = "watertank"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 5000, MAT_GLASS = 3000)
+	build_path = /obj/item/weapon/watertank/atmos
+	category = list("initial", "Atmos")
+
+/datum/design/chemtank
+	name = "Chemical storage tank"
+	id = "chemtank"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 10000, MAT_GLASS = 10000)
+	build_path = /obj/structure/reagent_dispensers
+	category = list("initial", "Atmos")
+
+/datum/design/pneumaticcannon
+	name = "Pneumatic cannon"
+	id = "pneumaticcannon"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 10000)
+	build_path = /obj/item/weapon/pneumatic_cannon
+	category = list("hacked", "Atmos")
+
+/datum/design/rpd
+	name = "Rapid pipe dispenser (RPD)"
+	id = "rpd"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 75000, MAT_GLASS = 37500)
+	build_path = /obj/item/weapon/pipe_dispenser
+	category = list("hacked", "Atmos")
```

Comment:
You realize you made the better cannon easier to build than the ghetto cannon right